src/components/Tool/FixCommitmentInfo/Contact.test.tsx

```diff
@@ -104,44 +104,8 @@ describe('FixCommitmentContact', () => {
     expect(setContactFocus).toHaveBeenCalledWith(testData.id, TabKey.Donations);
   });
 
-  it('should fail validation', async () => {
```

Comment:
Should we be removing this? Seems like it's a good test to have in place to ensure the status is filled in before the form can be submitted

Reply:
For this tool we actually want users to be able to change certain fields without changing all the fields. So we want them to be able to leave some fields blank. I'll remove `required` from the yup.